test/util/test_path.cpp

```diff
@@ -292,4 +294,21 @@ std::string TestDirNameGenerator::next()
     return m_path + "/" + std::to_string(m_counter++);
 }
 
+std::shared_ptr<DB> get_test_db(const std::string& path, const char* crypt_key)
+{
+    const char* str = getenv("UNITTEST_LOG_LEVEL");
+    realm::util::Logger::Level core_log_level = realm::util::Logger::Level::off;
+    if (str && strlen(str) != 0) {
+        std::istringstream in(str);
+        in.imbue(std::locale::classic());
+        in.flags(in.flags() & ~std::ios_base::skipws); // Do not accept white space
+        in >> core_log_level;
+    }
+
+    DBOptions options;
+    options.logger = std::make_shared<util::StderrLogger>(core_log_level);
```

Comment:
Should this use the same logger from `TestContext`? (e.g. timestamps or configured to log to a file instead of stderr)

Reply:
I think we should use different loggers. We want the logger in `TextContext` to log at `info` level, but in general we don't want core log output.